Packs/PhishLabs/Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.py

```diff
@@ -27,6 +27,13 @@
 
 
 class Client(BaseClient):
+
+    def __init__(self, base_url, verify=True, proxy=False, auth=None, reliability='B - Usually reliable'):
+
+        self.reliability = reliability
+
+        BaseClient.__init__(self, base_url=base_url, verify=verify, proxy=proxy, auth=auth)
```

Comment:
It is more common to start with the base init.
```suggestion
        BaseClient.__init__(self, base_url=base_url, verify=verify, proxy=proxy, auth=auth)
        self.reliability = reliability
```